src/styles/LoadingComponent.css

```diff
@@ -1,8 +1,6 @@
 .loading-component {
   width: 100%;
-  height: 100%;
   border-radius: 5px;
-  margin: 25% auto;
```

Comment:
This height and margin were forcing the loading spinner screen to have a scroll bar. Height 100% for the loading component is in addition to the height of the nav bar, hence scroll bar. Margin 25% refers to a top/bottom margin of 25% or view WIDTH, which also created a scroll bar for some screen sizes (short wide ones). I could remove these vertical positioning specs because I switched the spinner to absolute positioning instead of relative to this component.